src/mtx3/constants/index.ts

```diff
@@ -5,3 +5,4 @@
  */
 
 export * from '../../mtx/constants/index.js';
+export * as baudRates from './baudRates.js';
```

Comment:
```suggestion

export * as baudRates from './baudRates.js';
```

to separate mtx1 block from mtx3

Reply:
applied